omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java

```diff
@@ -13,14 +13,6 @@
  */
 package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_9;
 
-import static org.junit.Assert.assertEquals;

```

Comment:
You didn't make any changes to this file, I would say revert these formatting changes too